bgpd/bgp_vty.c

```diff
@@ -6414,10 +6452,16 @@ DEFUN (clear_bgp_ipv6_safi_prefix,
        "IPv6 prefix\n")
 {
 	int idx_safi = 3;
-	int idx_ipv6_prefixlen = 5;
+	int idx_ipv6_prefix = 5;
+	safi_t safi = SAFI_UNICAST;
+
+	argv_find_and_parse_safi(argv, argc, &idx_safi, &safi);
+	if (argv_find(argv, argc, "prefix", &idx_ipv6_prefix))
+		idx_ipv6_prefix += 1;
```

Comment:
the whole idea of `argv_find` (and `DEFPY`) is to avoid manual index calculations, here you can simply do

```
int idx = 0;
...
const char *prefix = argv_find(argv, argc, "X:X::X:X/M", &idx) ? argv[idx]->arg : NULL;
```

Reply:
ok